Chummer/Classes/clsXmlManager.cs

```diff
@@ -1857,6 +1857,69 @@ public bool Equals(CustomDataDirectoryInfo other)
             return other != null && Name == other.Name && Path == other.Path;
         }
 
+        /// <summary>
+        /// Reads in an Manifest.xml in the given directory and returns it's language specific description. Defaults to an description without any attributes.
+        /// </summary>
+        /// <param name="strDirectory"></param>
+        /// <returns>A description as String, adds a comment if it defaulted to a node without language attribute</returns>
+        public string GetManifestData(string strDirectory)
+        {
+            string strLanguage = GlobalOptions.Language;
+            XmlDocument xmlObjManifest = new XmlDocument();
+            string strFullDirectory = strDirectory + "\\manifest.xml";
```

Comment:
Rather than using escaped characters, it's probably better to use `Path`'s functions. Should also help you avoid the need for `@` flags everywhere, too.